src/utils.ts

```diff
@@ -95,3 +95,42 @@ export function createJavaScriptFromObject(obj_: any, intend = false) {
     }
     return final.join("\n");
 }
+
+export type Tree = Record<string, Function | any> | undefined | null;
+export type TreeFilter = string | ((tree: Tree) => boolean | Boolean); // whats the difference :)
+
+export function findInTree(
+    tree: Tree,
+    searchFilter: TreeFilter,
+    args: { walkable?: string[]; ignore?: string[]; maxRecursion: number } = { maxRecursion: 100 },
+): Tree | null | undefined {
+    const { walkable, ignore, maxRecursion } = args;
+
+    if (maxRecursion <= 0 || !tree) return undefined;
+
+    // @ts-ignore
+    // yeah I get all expressions arent callable. that's why IM CHECKING IF IT CAN BE.
+    if ((typeof searchFilter === "string" && tree?.[searchFilter]) || searchFilter(tree))
```

Comment:
You aren't actually checking it, remove that `// @ts-ignore`

```ts
if ((typeof searchFilter === "string" && typeof tree?.[searchFilter] === "function") || (typeof searchFilter === "function" && searchFilter(tree)))
```

Reply:
done in 59bb527f4c86af6e832889f3daccc8260b784d21